diagnostic.md

```diff
@@ -58,18 +58,18 @@ What does typing `bin/rails dbconsole` in a directory that houses a rails app do
 us?
 
 ```md
-// your response here
+This will start rails in the terminal.
```

Comment:
This is a typo and should have been `bin/rails console` so no points will be deducted for this.